### PR TITLE
Fix starting with multiple schedulers with --schedulers

### LINKED ISSUE #201: [BUG] Fail to start Mars with multiple schedulers specified with --schedulers
**Describe the bug**
Fail to start Mars with multiple schedulers specified with --schedulers and separated with commas. Error reported as

```
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 766, in gevent._greenlet.Greenlet.run
  File "mars/actors/pool/gevent_pool.pyx", line 444, in mars.actors.pool.gevent_pool.ActorRemoteHelper._send_remote
  File "mars/actors/pool/gevent_pool.pyx", line 448, in mars.actors.pool.gevent_pool.ActorRemoteHelper._send_remote
  File "mars/actors/pool/gevent_pool.pyx", line 437, in mars.actors.pool.gevent_pool.ActorRemoteHelper._new_connection
  File "mars/actors/pool/gevent_pool.pyx", line 442, in mars.actors.pool.gevent_pool.ActorRemoteHelper._new_connection
  File "mars/actors/pool/gevent_pool.pyx", line 365, in mars.actors.pool.gevent_pool.Connections.connect
  File "mars/actors/pool/gevent_pool.pyx", line 380, in mars.actors.pool.gevent_pool.Connections.connect
  File "/Users/wenjun.swj/miniconda3/lib/python3.7/site-packages/gevent/socket.py", line 80, in create_connection
    host, port = address
ValueError: not enough values to unpack (expected 2, got 1)
```

**To Reproduce**
Start local Mars cluster with

```bash
mars-scheduler -H 127.0.0.1 --level debug -p 13524 --schedulers localhost:13524,localhost:13525
mars-scheduler -H 127.0.0.1 --level debug -p 13525 --schedulers localhost:13524,localhost:13525
mars-worker -a 127.0.0.1 --cpu-procs 1 --level debug --cache-mem 16m --schedulerslocalhost:13524,localhost:13525 --ignore-avail-mem
mars-worker -a 127.0.0.1 --cpu-procs 1 --level debug --cache-mem 16m --schedulerslocalhost:13524,localhost:13525 --ignore-avail-mem
```

### PR DESCRIPTION
**Pull request by wjsi**
## What do these changes do?

Fix starting with multiple schedulers with ``--schedulers`` arg, also add cases for multiple schedulers with and without etcd. Remove profile in ``base_app`` as it is useless under multiple process scenarios. 

## Related issue number

Fixes #201

### CODE REVIEW

File: mars/base_app.py [hunk 1]
```diff
@@ -185,16 +183,7 @@ def create_pool(self, *args, **kwargs):
         return create_actor_pool(*args, **kwargs)
 
     def main_loop(self):
-        if self.args.profile == -1:
```
Comment (chineking) [inline mars/base_app.py]: Why remove the profile?

File: mars/core.py [hunk 2]
```diff
@@ -27,6 +27,8 @@
 from .tiles import Tilesable, handler
 from .graph import DAG
 
+_sorted_slots = dict()
```
Comment (chineking) [inline mars/core.py]: Maybe treating it as a class property of `Base` is a better solution, it looks a bit weird here.

File: mars/base_app.py (hunk 1 above)
Comment (wjsi) [reply mars/base_app.py]: Profile config is useless in main process.

File: mars/core.py (hunk 2 above)
Comment (wjsi) [reply mars/core.py]: Use __key_{cls.__name} to store slots for every class.

**Pull request merged**